digital_benchmark/digital_benchmark/settings.py

```diff
@@ -118,3 +119,11 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# facebook_benchmark settings
+facebook_graph_api_version = '3.1'
```

Comment:
use ALL_CAPS convention for constants (matching the rest of the constant names)

Reply:
ok